features/partner/partner_assessment.feature

```diff
@@ -56,9 +56,9 @@ Scenario: An applicant and partner's combined capital is over the lower threshol
       |     C     |  2022-08-22  | 500.50 |       0           | 75.00 |       15.0         |        410.5          |
       |     C     |  2022-09-22  | 500.50 |       0           | 75.00 |       15.0         |        410.5          |
     When I retrieve the final assessment
-    Then I should see the following "overall_disposable_income" details:
-      | attribute                    | value    |
-      | total_disposable_income      | 354.09   |
+    Then I should see the following "disposable_income_summary" details:
+      | attribute                             | value    |
+      | combined_total_disposable_income      | 354.09   |
```

Comment:
There was an interesting little bug here - we were listing the partner allowance in the applicant outgoings section of the response, but including it in the disposable income calculation of the _partner_. This didn't matter for CCQ because when we added everything together the sums worked out fine, and CCQ didn't display the individual person's disposable income totals, but refactoring for this PR caused the bug to be fixed, so this test has to change accordingly.